src/interfaces/store-interfaces/state-interfaces/name-request-interface.ts

```diff
@@ -28,16 +28,16 @@ export interface NameRequestApplicantIF {
 export interface NameRequestIF {
   nrNumber: string
   entityType: CorpTypeCd
-  details: NameRequestDetailsIF | {}
-  applicant: NameRequestApplicantIF | {}
+  details: NameRequestDetailsIF
+  applicant: NameRequestApplicantIF
   filingId: number
 }
 
 // NB: use cloneDeep when assigning EmptyOrgPerson
 export const EmptyNameRequest: NameRequestIF = {
   nrNumber: '',
   entityType: null,
-  details: {},
-  applicant: {},
+  details: {} as NameRequestDetailsIF,
+  applicant: {} as NameRequestApplicantIF,
```

Comment:
These should probably be null and handled accordingly in code but that's out of scope for this cleanup.

What I did here is remove the `{}` union type from `details` to pre-empt an error where `details.approvedName` was not recognized as valid.

As far as I can tell, the code runs normally atm.